rules/do-you-edit-your-videos-for-tiktok/rule.md

```diff
@@ -0,0 +1,79 @@
+---
+type: rule
+title: TikTok - Do you edit your videos for TikTok?
+uri: do-you-edit-your-videos-for-tiktok
+authors:
+  - title: Jonty Gardner
+    url: https://www.ssw.com.au/people/jonty-gardner
+    img: https://www.ssw.com.au/people/static/Jonty-Gardner-Profile-f8b9960c1c5482051abe7255cbc2dfcd.jpg
+created: 2022-05-02T05:25:08.857Z
+guid: efdbe798-d571-425d-a13c-6768b1b3d747
+---
+TikTok is a great way to engage with your audience via social media. TikTok is a more fun, informal, and relaxed social media platform compared to the likes of YouTube, Facebook, LinkedIn etc. It allows for a different style of creative freedom and spontaneity in your videos. 
+
+However, to truly engage effectively with your audience, it is important to format and edit your videos with some rules and best practices that are optimised for TikTok.  Doing this will ensure your videos are seen exactly how you intended them to be seen.
```

Comment:
Change from:
However, to truly engage effectively with your audience, it is important to format and edit your videos with some rules and best practices that are optimised for TikTok.  Doing this will ensure your videos are seen exactly how you intended them to be seen.
To:
However, to effectively engage with your audience, it is important to format and edit your videos following editing best practices that are optimised for TikTok.  Doing this will ensure your videos are noticed exactly by whom and how you intended them to be seen.